src/objective-c/examples/tvOS-sample/tvOS-sample/Info.plist

```diff
@@ -3,7 +3,7 @@
 <plist version="1.0">
 <dict>
 	<key>CFBundleDevelopmentRegion</key>
-	<string>$(DEVELOPMENT_LANGUAGE)</string>
+	<string>en</string>
```

Comment:
Why is this different from interceptor sample (en_US)?

Reply:
I think `en_US` was chosen from a drop-down menu of Xcode's plist editor. `en` is just more universal. I remember typing these two in as opposed to selecting from a drop-down menu. Replacing `$(DEVELOPMENT_LANGUAGE)` with `en` is just because Bazel complains about this being `undefined`. Is it better make all of them the same?

Reply:
Yea I think it makes sense to make all of them the same. Or simply just remove this key/value pair. I don't think it is essential for Bazel (we do for internal projects).